packages/@aws-cdk/aws-cloudfront/test/test.basic.ts

```diff
@@ -322,4 +323,168 @@ export = {
     test.done();
   },
 
+  'distribution with resolvable lambda-association'(test: Test) {
+    const stack = new cdk.Stack();
+    const sourceBucket = new s3.Bucket(stack, 'Bucket');
+
+    const lambdaFunction = new lambda.Function(stack, 'Lambda', {
+      code: lambda.Code.inline('foo'),
+      handler: 'index.handler',
+      runtime: lambda.Runtime.Nodejs810
+    });
+
+    const lambdaVersion = new lambda.Version(stack, 'LambdaVersion', {
+      lambda: lambdaFunction
+    });
+
+    new CloudFrontWebDistribution(stack, 'AnAmazingWebsiteProbably', {
+      originConfigs: [
+        {
+          s3OriginSource: {
+            s3BucketSource: sourceBucket
+          },
+          behaviors: [
+            {
+              isDefaultBehavior: true,
+              lambdaFunctionAssociations: [{
+                eventType: LambdaEdgeEventType.OriginRequest,
+                lambdaFunction: cdk.Token.asAny(lambdaVersion.versionArn)
+              }]
+            }
+          ]
+        }
+      ]
+    });
+
+    expect(stack).toMatch({
```

Comment:
Please change this assertion to use `to(haveResource(...))` (as in other tests) to avoid asserting too much about the state of the template.

Only test for the fields you're interested in in this test.

Reply:
@rix0rrr I have to update the imports in this case, since there are other test which use the `toMatch` pattern. I guess they are a bit older and I have no clue what are the relevant resources for that are, so I cannot rewrite all tests.

For my test I'll now get this error when executing the test:

```
@aws-cdk/aws-cloudfront: expect.extend({
@aws-cdk/aws-cloudfront: ^
@aws-cdk/aws-cloudfront: ReferenceError: expect is not defined
@aws-cdk/aws-cloudfront:     at Object.<anonymous> (/home/markusw/source/aws-cdk/packages/@aws-cdk/assert/jest.ts:24:1)
@aws-cdk/aws-cloudfront:     at Module._compile (internal/modules/cjs/loader.js:759:30)
@aws-cdk/aws-cloudfront:     at Module.replacementCompile (/home/markusw/source/aws-cdk/tools/cdk-build-tools/node_modules/append-transform/index.js:58:13)
@aws-cdk/aws-cloudfront:     at Module._extensions..js (internal/modules/cjs/loader.js:770:10)
@aws-cdk/aws-cloudfront:     at Object.<anonymous> (/home/markusw/source/aws-cdk/tools/cdk-build-tools/node_modules/append-transform/index.js:62:4)
@aws-cdk/aws-cloudfront:     at Module.load (internal/modules/cjs/loader.js:628:32)
@aws-cdk/aws-cloudfront:     at Function.Module._load (internal/modules/cjs/loader.js:555:12)
@aws-cdk/aws-cloudfront:     at Module.require (internal/modules/cjs/loader.js:666:19)
@aws-cdk/aws-cloudfront:     at require (internal/modules/cjs/helpers.js:16:16)
@aws-cdk/aws-cloudfront:     at Object.<anonymous> (/home/markusw/source/aws-cdk/packages/@aws-cdk/aws-cloudfront/test/test.basic.ts:2:1)
```
Is there something to change on the project to use expect (from jest)?

Reply:
This module doesn't seem to be using `jest` though. 

Yes, you have to update the exports, you have to add `haveResource` to the import from `@aws-cdk/assert` and take nothing away.

Reply:
:+1: